### PR TITLE
Fixing few minor MP deps issues

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Review by mkouba — APPROVED** [review]
Looks good

File: fractions/microprofile/microprofile/pom.xml [hunk 1]
```diff
@@ -125,4 +121,21 @@
     </dependency>
   </dependencies>
 
+  <profiles>
```
Comment (kenfinnigan) [inline fractions/microprofile/microprofile/pom.xml:124]: Why was this added?

The entirety of MP should be part of product

Comment (mkouba) [reply fractions/microprofile/microprofile/pom.xml:124]: Hm, sorry for wrong approval. Now I'm a little bit confused. What MP do we acutally implement in community/product? When looking at MP APIs versions it's like a mix of MicroProfile 1.4 (config, jwt) and 1.3 (openapi, rest-client, opentracing, FT, health, metrics).

Comment (kenfinnigan) [reply fractions/microprofile/microprofile/pom.xml:124]: Technically it's MP 1.3, but some of the sub specs are actually further along
